### ISSUE TITLE
Lacking test for imports needed above recipy

### ISSUE THREAD

**Issue by oew1v07** [recipy/recipy#113]
So from __future__ import division has to go at the top of a python file. Will recipy work with this under every scenario? I think there should be a test for this.

**Comment by robintw** [comment]
I agree - this is a simple test that could be implemented as part of https://github.com/softwaresaved/recipy-test.
